index.js

```diff
@@ -94,9 +94,13 @@ module.exports = (context, options) => {
 
     // Cherry-pick lodash modules for smaller bundles
     if (options.lodash) {
+        const defaultLodashOptionsIds = ['lodash/fp'];
```

Comment:
Looking at https://github.com/lodash/babel-plugin-lodash/blob/master/src/config.js#L17, I think that when we pass `id`, it completely overrides the default ones. This means that we must include all those defaults.

Reply:
It has been reported in this issue. https://github.com/lodash/babel-plugin-lodash/issues/184 .
We can re-include the defaults, but we should be concerned about future fixes on this matter.